docs/index.md

```diff
@@ -0,0 +1,13 @@
+# LeanCode.ContractsGenerator
+
+This is a C#-based contracts generator that operates on LeanCode's notion of [CQRS-like API]. It allows you to generate
+strongly-typed contracts & clients in a number of languages based on plain old C# classes.
+
+## Documentation
+
+1. [Tutorial](./tutorial.md),
+1. [Types](./types.md),
+1. [Serialization](./serialization.md),
+1. [Guidelines](./guidelines.md).
```

Comment:
I know markdown renders it as consecutive numbers anyways, but I guess we want to number it manually regarless?

Reply:
TBH I'm not sure :upside_down_face:

Reply:
Keeping all as `1` makes future diffs less awkward :slightly_smiling_face: